onnxruntime/core/graph/model.cc

```diff
@@ -738,6 +739,36 @@ Status Model::Load(int fd, const PathString& model_path, std::shared_ptr<Model>&
   return Status::OK();
 }
 
+// static
+common::Status Model::LoadFromModelEditorApiModel(const OrtModel& model_editor_api_model,
+                                                  const IOnnxRuntimeOpSchemaRegistryList* local_registries,
+                                                  const ModelOptions& options,
+                                                  const logging::Logger& logger,
+                                                  std::unique_ptr<Model>& model) {
+  model = std::make_unique<Model>();
```

Comment:
>model = std::make_unique<Model>(); [](http://example.com/codeflow?start=2&length=34)

In the view of the fact, that the output is populated right away with potentially incomplete model, I suggest that we make return value [[nodiscard]]

Reply:
Isn't `Status` declared nodiscard so that's automatically done already?

https://github.com/microsoft/onnxruntime/blob/5be82eb7b2b48515eff5d12506bc92f4cd061b3e/include/onnxruntime/core/common/status.h#L113

Reply:
Ok.